src/fetchMovies.js

```diff
@@ -31,7 +31,7 @@ export async function specMovie(movieId) {
     return modifiedObject;
   }
   catch {
-    const notFound = 'Page not found'
+    const notFound = '404 (Not found)'
```

Comment:
I have two comments on this line:

First, the variable serves no purpose. If you just want to return the string `"404 (Not found)"`, you can just do `return "404 (Not found)` without storing it in a temporary variable.

But I would not return such a string to begin with. The reason is that the string is very "human readable", but the purpose of returning a string here is not so that it is read by the user, but to indicate an error. I would return `null` instead.